src/MultiField/MultiField.jl

```diff
@@ -15,12 +15,21 @@ using Gridap.TensorValues
 using Gridap.CellData
 using Gridap.Fields
 
+using Gridap.FESpaces: FEBasis, TestBasis, TrialBasis, get_cell_dof_values
```

Comment:
I think the changes in this file are no longer needed after moving the code to `CellFields.jl`. Can you revert them?